### PR TITLE
Admin UI

### PR DESCRIPTION
**Pull request by StMarian**
Also in this PR:
- database changes
- EADesktopManager structure

### CODE REVIEW

File: Database/Creation/SetupDB.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+IF DB_ID (N'EADB') IS NOT NULL
+BEGIN
+	ALTER DATABASE EADB SET SINGLE_USER WITH ROLLBACK IMMEDIATE
+	DROP DATABASE EADB
+END
+GO
+
+CREATE DATABASE EADB
+GO
```
Comment (Kp0c) [inline Database/Creation/SetupDB.sql]: Empty line at EOF)) but nevermind

File: Database/SPs/GetUserApp.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
+USE EADB
+GO
+
+IF OBJECT_ID('GetUserApp', 'P') IS NOT NULL
+	DROP PROCEDURE dbo.GetUserApp
+GO
+
+CREATE PROCEDURE dbo.GetUserApp
+(
+	@UserID	INT
```
Comment (Kp0c) [inline Database/SPs/GetUserApp.sql]: I think get user apps by username more common operation

File: EasyAccess/EAService/EAService.cs [hunk 3]
```diff
@@ -39,7 +39,6 @@ public void Log(string log)
                     EventLog.CreateEventSource("EAService", "EAService");
                 }
 
-                eventLog.Source = "EAService";
```
Comment (Kp0c) [inline EasyAccess/EAService/EAService.cs]: just removed?
I suggest do all this stuff on initializing service

Comment (StMarian) [reply EasyAccess/EAService/EAService.cs]: Moved to Designer file

File: Database/SPs/GetUserApp.sql (hunk 2 above)
Comment (StMarian) [reply Database/SPs/GetUserApp.sql]: I binded to Id so I used it.

File: Database/Creation/SetupDB.sql (hunk 1 above)
Comment (StMarian) [reply Database/Creation/SetupDB.sql]: Forgot here)

**Pull request merged**